cohorts/cohort.py

```diff
@@ -940,15 +937,18 @@ def load_single_patient_isovar(self, patient, variants, epitope_lengths):
         #           123456789AB
         # AAAAAAAAAAVAAAAAAAAAA
         protein_sequence_length = (max(epitope_lengths) * 2) - 1
-        df_isovar = variants_to_protein_sequences_dataframe(
+        allele_reads_generator = reads_overlapping_variants(
             variants=variants,
             samfile=rna_bam_file,
+            min_mapping_quality=1)
+        protein_sequences_generator = reads_generator_to_protein_sequences_generator(
+            allele_reads_generator,
             protein_sequence_length=protein_sequence_length,
-            min_reads_supporting_rna_sequence=3, # Per Alex R.'s suggestion
-            min_transcript_prefix_length=MIN_TRANSCRIPT_PREFIX_LENGTH,
-            max_transcript_mismatches=MAX_REFERENCE_TRANSCRIPT_MISMATCHES,
+            # Per Alex R.'s suggestion; equivalent to min_reads_supporting_rna_sequence previously
+            min_variant_sequence_coverage=3,
```

Comment:
Revised opinion is that it's probably safe to use 2 here.

(Unsourced intuition)